### PR TITLE
Update in.m3u

### PR DESCRIPTION
**Pull request by desiflex**
Newdons Tv

### CODE REVIEW

File: channels/in.m3u [hunk 1]
```diff
@@ -11,6 +11,10 @@ http://103.199.161.254/Content/7smusic/Live/Channel(7smusic)/index.m3u8
...
+https://video-weaver.maa01.hls.ttvnw.net/v1/playlist/CrgFG968dAy-iSQltxXB3lsgBtnO1zu2KQrfK3cGuHfabQAMGRPotoKhio0ElzuriYxH0USwnm9ZfDXMr1lt9gq1V81MKfrijHIaZMGyH_9XqgGV-7E9pG2Q8siVbnmSDuvsTCo4QEvV9HzTaShHsuHET4SENXa-QY4dMBUhO_5lqo37ktzW5ICJRIcqDRDYF_bc5fKd3Si_lGcti9n_oOq4FEoifSjbZX5ckUMwYUKrfTSLrYQKyjaTm107zNZnSDLEI2JTCmwnwtDP0LoakFKBCv5g0De2dtY85EWIuvIAYp0y6tBvkXgMd7G4-Hm9U8WdLA86HdVLLwGBY0snt2DZW-Er8e867ueUBwJeCpIS2OFhhuNNWpA_i6Z8KSGxfSjWQGm_VbqdQFjOs3akUkuv4lcQ34Mr3amhWZBvA3pD_g6J_P6Gt8EHqR_irbikzqdkvob9aQEll7o1bOtghCiFty5pmEjlbGY-0sdYcG1uxhXlpUGD6kO1tl6LA7Wb83C30L6OxK1rqavkINZDfh6xwkcMO547cavpXtRR9iLMQdP6vjR8sSr3N49TTsG1dx9HaMeLAH2nVPFkLJZnmdqYZe-4hvV_e-4sDie-uP_XXNv9kakEad68eRx_dBFB9U-1IG2GldNJcsXEZMN1GFeKSWxSrUDUs_FIlCksAxXwgAakii-kwgHkf7c2ZHdzI9Z6ZWVpceKBO7ORpr3R_N2hkcQDg1zbPPI53YCv7e3Jn8GUYfRJ8zqChQbJJQlGGuRHGvcxjsW8y-14NgWwXPYYOHaHMyebB_LZ1R7vJ-TRk_GJRvPqR3N1h8OqYALx5JCmtTvHdq7M7Ci8QuUnXkHFqiXuq9hPqcXFD3Eu0zsFWjhbE9MGIRNBKL9nyTcWdpL37CKFvDeifvSbGSxOuVfjvOBntX6j6kPsEhD_ZvHY_X5xytAnkNDRm6YkGgySVeNRdfNR6OaqZJw.m3u8
```
Comment (themak1985) [inline channels/in.m3u:17]: This is not a valid link.

**Review by themak1985 — CHANGES_REQUESTED** [review]

File: channels/in.m3u (hunk 1 above)
Comment (MapGuy11) [reply channels/in.m3u:17]: 2nd one doesn't work ERROR 404 NOT FOUND

**Review by MapGuy11 — DISMISSED** [review]
2nd one doesn't work 404 not found.

File: channels/in.m3u (hunk 1 above)
Comment (LaneSh4d0w) [reply channels/in.m3u:17]: Pasting a Twitch playlist link is not useful. You can use the Streamlink query hosts if you want to use a Twitch link.

**Review by themak1985 — APPROVED** [review]

**Comment by themak1985** [comment]
@desiflex you can always modify a pull request - you do not need to keep submitting new pull requests. I have gone ahead and removed the bad link you added and kept the good one. @LaneSh4d0w @MapGuy11 once you approve, we can merge.

**Review by BellezaEmporium — APPROVED** [review]

**Comment by freearhey** [comment]
- replaced the short link with the original to remove unnecessary redirects
- corrected the name of the language

**Review by freearhey — APPROVED** [review]

**Pull request merged**